packages/analytics/lib/index.d.ts

```diff
@@ -229,6 +229,7 @@ export namespace FirebaseAnalyticsTypes {
      */
     value: number;
   }
+
```

Comment:
there are still spacing changes all over, expanding the diff for no functional purpose 🤔

Reply:
Unfortunately, it's not trivial to remove these spaces that prettier added without going line by line over hundreds of lines of code over the entire PR. If you have any ideas on how to achieve this without me having to spend literal hours removing spaces, let me know :)